client/question/services/FeedbackGeneratorService.js

```diff
@@ -257,40 +322,87 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     /**
+     * Concats two strings to create testCaseKey
+     *
+     * @param {string} testSuiteId
+     * @param {string} testCaseIndex
+     */
+    var _getTestCaseKey = function(testSuiteId, testCaseIndex) {
+      return testSuiteId + '-' + testCaseIndex;
+    };
+
+    /**
      * Returns the Feedback object created as a result of a specified
      * correctness test.
      *
-     * @param {string|null} outputFunctionName If needed, is string of the
-     *    name for the function needed to process/format the user's output.
      * @param {CorrectnessTest} TestCase
+     * @param {number} testCaseIndex
      * @param {*} observedOutput Actual output for running user's code.
+     * @param {string} testSuiteId
```

Comment:
Optional: I'd like to suggest putting this just before testCaseIndex in the argument ordering, since logically the suite comes before the test case.

Reply:
Changed